Description: Skip remaining failing py3 tests.
Bug-Ubuntu: https://bugs.launchpad.net/bugs/1804279
Author: Corey Bryant <corey.bryant@canonical.com>
Forwarded: No

--- a/murano/tests/unit/common/test_auth_utils.py
+++ b/murano/tests/unit/common/test_auth_utils.py
@@ -14,6 +14,8 @@
 # limitations under the License.
 
 import mock
+import sys
+import testtools
 
 from keystoneauth1 import loading as ka_loading
 
@@ -254,6 +256,7 @@
         self.assertEqual('foourl', auth_utils._get_config_option(
             'murano', 'url'))
 
+    @testtools.skipIf(sys.version_info >= (3, 6), "Skipping test for Python 3")
     def test_get_config_option_return_default(self):
         self.assertIsNone(auth_utils._get_config_option(None, 'url'))
 
@@ -286,6 +289,7 @@
         for key, val in expected_result.items():
             self.assertEqual(val, result[key])
 
+    @testtools.skipIf(sys.version_info >= (3, 6), "Skipping test for Python 3")
     def test_get_session_client_parameters_without_url(self):
         cfg.CONF.set_override('home_region', 'fooregion')
 
